package.json

```diff
@@ -4,6 +4,10 @@
   "description": "Digital Marketplace",
   "main": "index.js",
   "scripts": {
+    "unscored": "docker exec dm_db psql -v ON_ERROR_STOP=1 -U digitalmarketplace digitalmarketplace -f /workspace/cypress/fixtures/dbReset.sql -f /workspace/cypress/fixtures/users.sql -f /workspace/cypress/fixtures/swuOpportunityPublished.sql -f /workspace/cypress/fixtures/swuProposalUnscored.sql -f /workspace/cypress/fixtures/myAccount.sql",
+    "tqscored": "docker exec dm_db psql -v ON_ERROR_STOP=1 -U digitalmarketplace digitalmarketplace -f /workspace/cypress/fixtures/dbReset.sql -f /workspace/cypress/fixtures/users.sql -f /workspace/cypress/fixtures/swuOpportunityPublished.sql -f /workspace/cypress/fixtures/swuProposalScoredTQ.sql -f /workspace/cypress/fixtures/myAccount.sql",
+    "ccscored": "docker exec dm_db psql -v ON_ERROR_STOP=1 -U digitalmarketplace digitalmarketplace -f /workspace/cypress/fixtures/dbReset.sql -f /workspace/cypress/fixtures/users.sql -f /workspace/cypress/fixtures/swuOpportunityPublished.sql -f /workspace/cypress/fixtures/swuProposalScoredCC.sql -f /workspace/cypress/fixtures/myAccount.sql",
+    "tsscored": "docker exec dm_db psql -v ON_ERROR_STOP=1 -U digitalmarketplace digitalmarketplace -f /workspace/cypress/fixtures/dbReset.sql -f /workspace/cypress/fixtures/users.sql -f /workspace/cypress/fixtures/swuOpportunityPublished.sql -f /workspace/cypress/fixtures/swuProposalScoredTS.sql -f /workspace/cypress/fixtures/myAccount.sql",
```

Comment:
These are just convenience scripts for testing that the change works (need to check that status changing to "not awarded" regardless of how far a proposal made it through the process). I'll pull them out after testing